crates/cairo-addons/src/vm/vm_consts.rs

```diff
@@ -147,6 +147,15 @@ fn create_var_type(
                     CairoVarType::Struct { name: base_type.to_string(), members, size }
                 }
                 None => {
+                    if base_type == "(fp_val: felt, pc_val: felt*)" {
+                        // Manual handling of fp_val and pc_val, which causes issues in both VMs.
```

Comment:
what kind of issue ?

Reply:
in RustVM it just tells you that (fp_al: felt, pc_val: felt*) cannot be bound to a type (and thus it crashes at this step, when loading the vm consts)
in the PythonVM it raises an exception upon trying to access fp_and_pc (same kind of issues)